src/vaishnavi/A12_P2_MaxPrimeNumber.java

```diff
@@ -0,0 +1,39 @@
+package vaishnavi;
+
+import java.util.Scanner;
+
+public class A12_P2_MaxPrimeNumber {
+	int primeNumber;
+	boolean isPrimeNumber(int num) {
+		boolean flag = true;
+		for(int index = 2;index<=num/2;index++) {
+			if(num%index==0) {
+			return flag = false;
+			}
+		}
+			return flag = true;
+	}
+	void findMaxPrimeNumber(int firstNum, int lastNum) {
+		int maxPrimeCount=0;
+		for(int index = firstNum; index <= lastNum; index++) {
```

Comment:
as we know the last num has the highest value always so you can iterate the loop in reverse order to get max prime num

Reply:
just FYI